<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapeWorksStudioApp</class>
 <widget class="QMainWindow" name="ShapeWorksStudioApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShapeWorksStudio</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/ShapeWorksStudio.qrc">
    <normaloff>:/Studio/Images/shapes-icon.png</normaloff>:/Studio/Images/shapes-icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
/*
QWidget#dockWidgetContents
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(225, 225, 225, 255), stop:1 rgba(195, 195, 195, 255));
border: 1px solid rgb(60, 60, 60);
border-radius: 3px;
}
*/


QWidget#dockWidgetContents
{
border: 1px solid rgb(80,80,80);
border-radius: 3px;
}

QWidget#statusbar
{

background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(225, 225, 225, 255), stop:1 rgba(195, 195, 195, 255));

}

QWidget#zoomWidget
{

background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(225, 225, 225, 255), stop:1 rgba(205, 205, 205, 255));
border: 1px solid rgb(60, 60, 60);
border-radius: 3px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="QVTKWidget" name="qvtkWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QScrollBar" name="vertical_scroll_bar">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
      <item>
       <widget class="QCheckBox" name="center_checkbox">
        <property name="text">
         <string>Center</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="zoomWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Zoom:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="thumbnail_size_slider">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>8</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="actionSaveProjectAs"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewFullScreen"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="controlsDockWidget_2">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="importPage">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tableWidget"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout2">
           <item>
            <widget class="QToolButton" name="deleteButton">
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Remove</string>
             </property>
             <property name="statusTip">
              <string>Remove images</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../Resources/ShapeWorksStudio.qrc">
               <normaloff>:/Studio/Images/remove.png</normaloff>:/Studio/Images/remove.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="addButton">
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Add</string>
             </property>
             <property name="statusTip">
              <string>Add images</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../Resources/ShapeWorksStudio.qrc">
               <normaloff>:/Studio/Images/add.png</normaloff>:/Studio/Images/add.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="groomPage">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="margin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>42</width>
     <height>42</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImportMode"/>
   <addaction name="actionGroomMode"/>
   <addaction name="actionActionOptimizeMode"/>
   <addaction name="actionAnalysisMode"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionViewFullScreen">
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionGroomMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/ShapeWorksStudio.qrc">
     <normaloff>:/Studio/Images/tools.png</normaloff>:/Studio/Images/tools.png</iconset>
   </property>
   <property name="text">
    <string>Groom</string>
   </property>
  </action>
  <action name="actionImportMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/ShapeWorksStudio.qrc">
     <normaloff>:/Studio/Images/data.png</normaloff>:/Studio/Images/data.png</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionActionOptimizeMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/ShapeWorksStudio.qrc">
     <normaloff>:/Studio/Images/gear2.png</normaloff>:/Studio/Images/gear2.png</iconset>
   </property>
   <property name="text">
    <string>Optimize</string>
   </property>
  </action>
  <action name="actionAnalysisMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/ShapeWorksStudio.qrc">
     <normaloff>:/Studio/Images/chart.png</normaloff>:/Studio/Images/chart.png</iconset>
   </property>
   <property name="text">
    <string>Analysis</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveProjectAs">
   <property name="text">
    <string>Save Project As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/ShapeWorksStudio.qrc"/>
 </resources>
 <connections/>
</ui>
